lib/SlackBot.js

```diff
@@ -209,6 +209,7 @@ function Slackbot(configuration) {
             static_dir = slack_botkit.config.webserver.static_dir;
 
         slack_botkit.config.port = port;
+        slack_botkit.config.ip = slack_botkit.config.ip || '127.0.0.1';
```

Comment:
Can you double check if `slack_botkit.config` exists. If it's not, it will crash eventually.